frontend/javascripts/test/api/api_volume_latest.spec.js

```diff
@@ -25,19 +25,20 @@ test("setActiveCell should set the active cell id", t => {
 });
 
 test("getAnnotationTool should get the current tool", t => {
-  const {api} = t.context;
+  const { api } = t.context;
```

Comment:
Did prettier not run for this file before? :thinking:

Reply:
I did that change manually since my linter in sublime complained :see_no_evil: With our project-config, eslint marks these only as a warning (which is why the CI passes).